x-pack/plugins/canvas/public/lib/run_interpreter.js

```diff
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { interpretAst } from '@kbn/interpreter/public';
+import { interpretAst } from 'plugins/interpreter/interpreter';
```

Comment:
Why this change?  When I search for `interpretAst` it looks like it's coming out of `kbn-interpreter/src/public` and `fromExpression` is coming out of `kbn-interpreter/src/common`.  Was there an issue importing it via `@kbn/interpreter/public`?